infra/build/functions/ood_upload_corpus.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+#
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+import base64
+import logging
+import os
+import pickle
+import subprocess
+import sys
+import uuid
+# import requests
+
+def execute_shell_command(command_args, check_return_code=True):
+  """ """
+  result = subprocess.run(
+      command_args,
+      check=check_return_code,
+      capture_output=True,
+      text=True
+  )
+
+  if result.stdout:
+      print("   STDOUT:")
+      print(result.stdout.strip())
+  if result.stderr:
+      print("   STDERR:")
+      print(result.stderr.strip())
+
+
+def install_requirements():
```

Comment:
This is doing something at runtime that should have been done earlier. Can we install these in whatever image you are using so we don't have to do it now.